pkgs/development/python-modules/HPI/default.nix

```
@@ -0,0 +1,47 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, appdirs
+, click
+, decorator
+, logzero
+, lxml
+, more-itertools
+, mypy
+, orjson
+, pandas
+, pytz
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "HPI";
+  version = "0.0.20200417";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-cozMmfBF7D1qCZFjf48wRQaeN4MhdHAAxS8tGp/krK8=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
```

```suggestion
  nativeBuildInputs = [
    setuptools-scm
  ];
```

In python-modules we do expand all lists.